lua/scissors/edit-popup.lua

```diff
@@ -162,8 +162,9 @@ function M.editInPopup(snip, mode)
 
 	local bufName, winTitle
 	if mode == "update" then
-		bufName = u.snipDisplayName(snip)
-		winTitle = (" Editing %q [%s] "):format(bufName, nameOfSnippetFile)
+		local displayName = u.snipDisplayName(snip)
+		bufName = ("Edit Snippet %s"):format(displayName)
```

Comment:
Could you for consistency change the `%s` to `%q` here? (this just adds quotation marks)

Reply:
Did I get your suggestion right?  You talking about the bufName, right?

Reply:
Yes. The comment is attached to line 166 where only one `%s` occurs. Though the GitHub UI does not convey that very well, sadly...